### PR TITLE
fix: solve subgraph urls

### PR DESCRIPTION
**Pull request by frost-ostrich**
# Description
What does this PR solve?

Fixes # (issue)

# Checklist
- [ ] Code follows the style guide
- [ ] I have performed a self-review of my own code
- [ ] New and existing tests pass locally
- [ ] This PR is targeting the correct branch

### CODE REVIEW

**Review by mountainpath9 — DISMISSED** [review]

**Comment by frost-ostrich** [comment]
@medariox I've updated the tlc subgraph urls on mainnet, when we interact with these urls through the bot, we use a slightly different url than the normal webapp

File: apps/automation-v2/src/config/prodnets.ts [hunk 1]
```diff
@@ -9,8 +9,8 @@ const TLC_BATCH_LIQUIDATE_CONFIG: TlcBatchLiquidateConfig = {
   ACC_LIQ_MAX_CHUNK_NO: 100,
   MIN_ETH_BALANCE_WARNING: parseEther('0.1'),
   GAS_LIMIT: 1_000_000n,
-  SUBGRAPH_URL: 'https://thegraph.com/hosted-service/subgraph/templedao/templedao-tlc-liquidations',
-  SUBGRAPH_ALCHEMY_URL: 'https://subgraph.satsuma-prod.com/a912521dd162/templedao/tlc-liquidations-mainnet/api',
+  SUBGRAPH_URL: 'https://api.thegraph.com/subgraphs/name/templedao/templedao-tlc-liquidations',
+  SUBGRAPH_ALCHEMY_URL: 'https://subgraph.satsuma-prod.com/a912521dd162/templedao/temple-v2-mainnet/api',
```
Comment (medariox) [inline apps/automation-v2/src/config/prodnets.ts:13]: Why aren't we using the TLC liquidations subgraph here?

Comment (frost-ostrich) [reply apps/automation-v2/src/config/prodnets.ts]: updated 👍🏼

**Review by medariox — APPROVED** [review]
LGTM 👍

**Pull request merged**